packages/astro/src/runtime/client/dev-toolbar/toolbar.ts

```diff
@@ -159,6 +159,7 @@ export class AstroDevToolbar extends HTMLElement {
 				opacity: 0;
 				transition: opacity 0.2s ease-in-out 0s;
 				pointer-events: none;
+				user-select: none;
```

Comment:
Just a small thing I noticed while working on this, it's technically possible for the tooltip on hover to get selected by users somehow.